src/libs/actions/Welcome/index.ts

```diff
@@ -48,6 +49,11 @@ function isOnboardingFlowCompleted({onCompleted, onNotCompleted, onCanceled}: Ha
             return;
         }
 
+        if (onboarding?.testDriveModalDismissed === false) {
```

Comment:
Why can't we simply do `!onboarding?.testDriveModalDismissed`?

Reply:
Got it, thank you for the explanation. Can you add that as a comment right before the condition?